esphome/components/sx1509/binary_sensor.py

```diff
@@ -0,0 +1,29 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome.components import binary_sensor
+from esphome.const import CONF_ID
+from esphome.components.sx1509.sensor import SX1509KeypadSensor, sx1509_ns
+
+CONF_ROW = 'row'
+CONF_COL = 'col'
```

Comment:
```suggestion
CONF_COLUMN = 'column'
```

long_and_verbose names